### PR TITLE
Send Testing Framework workflow message to Slack

### PR DESCRIPTION
**Pull request by lmilbaum**
1. Trigger the pipeline every hour instead of once a day
2. Publish the workflow results to Slack

### CODE REVIEW

**Comment by lmilbaum** [comment]
The CI failure is unrelated to the this PR

File: .github/workflows/testing-framework.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Testing Framework
 
 on:
   schedule: # schedule the job to run at 12 AM daily
-   - cron: '0 0 * * *'
+   - cron: '0 * * * *'
```
Comment (Gregory-Pereira) [inline .github/workflows/testing-framework.yaml:5]: I think this means it will run every hour. AFAIK `'0 0 * * *` is once a day.

Comment (lmilbaum) [reply .github/workflows/testing-framework.yaml:5]: I want this workflow to run once a hour

**Review by Gregory-Pereira — APPROVED** [review]
Optionally could remove the comment on line 4 but I leave that to your discretion.
/lgtm

**Comment by lmilbaum** [comment]
> Optionally could remove the comment on line 4 but I leave that to your discretion. /lgtm

Thanks for catching it.  Updated.

**Pull request merged**